### ISSUE TITLE
Still does not work with poetry 1.0.0b1

### ISSUE THREAD

**Issue by elendirx** [UnitedIncome/serverless-python-requirements#409]
Poetry 1.0.0b1 changed the behaviour of export.
This was fixed according to: https://github.com/UnitedIncome/serverless-python-requirements/pull/395

Except it wasn't.
The requirements.txt file is not saved in case there is no editable flag.

**Comment by gusostow** [comment]
Seconded

**Comment by billyshambrook** [comment]
Is there any movement on this?

Our current hack/workaround is to have the following in the serverless.yaml

```
plugins:
- serverless-python-requirements
- serverless-scriptable-plugin

custom:
  pythonRequirements:
    usePoetry: false
  scriptHooks:
    before:package:createDeploymentArtifacts: poetry export --without-hashes -f requirements.txt -o requirements.txt
```

This works so far, but does look messy in my yaml :(

Let me know if I can help!

**Comment by jimpudar** [comment]
Also hitting this issue

**Comment by BastienM** [comment]
Bumping this issue.
Just had the same problem.

**Comment by zepp-lee** [comment]
Same for me, please deploy the fix to the NPM registry asap to make the plugin less frustrating to work with for new users

**Comment by bsamuel-ui** [comment]
I'm closing this as resolved by #446
